packages/atlas/src/topic.ts

```diff
@@ -1,5 +1,6 @@
 import { Base } from "@/base";
 import { ITopic } from "@litespace/types";
+import { FindUserTopicsApiResponse } from "@litespace/types/dist/esm/topic";
```

Comment:
Don't import from `esm/` directly. You should import the module `ITopic.FindUserTopicsApiResponse`.